ColorsPanel.h

```diff
@@ -15,11 +15,10 @@ typedef struct ColorsPanel_ {
    Panel super;
 
    Settings* settings;
-   ScreenManager* scr;
 } ColorsPanel;
 
 extern const PanelClass ColorsPanel_class;
 
-ColorsPanel* ColorsPanel_new(Settings* settings, ScreenManager* scr);
```

Comment:
How much is this change related to the header format changes?

Reply:
Seems not at all, just cleanup while at it